quickget

```diff
@@ -810,6 +825,19 @@ function get_archlinux() {
     make_vm_config "${ISO}"
 }
 
+function get_cachyos() {
+    local HASH=""
+    local ISO=""
+    local URL=""
+    local VERSION=""
+
+    validate_release "releases_cachyos"
+    ISO="cachyos-cli-${RELEASE}-x86_64.iso"
+    URL="https://deac-fra.dl.sourceforge.net/project/cachyos-arch/cli-installer"
```

Comment:
I think this will not work:  you need to find a way to get and parse a mirror list or provide a static mirror URL  so that  the script can fetch the iso, ideally also a checksum file. If I run what the webget() will run here I get redirects and then an error. Also this script us used by a gui so needs to behave consistently for it.
And finally, given the aims of this project to be it would probably be advisable to include the gui-installer version as well as a release option.

Reply:
Yeah there is no api for that yet
but I will try to make one

Reply:
Hey @philclifford 

You can easily get the checksum from https://mirror.cachyos.org/ISO/cachyos-cli-2021.11.10-x86_64.iso
For example:  MD5=b944838bfea7ca9058ee599746f003d8d5330de813ce956dea2d9fd0d5a3545a

The next step is to create a router/loadbalancer for the mirros we have.

Reply:
Hi @ptr1337 if @vnepogodin updates the PR to set URL to "https://mirror.cachyos.org/ISO/" it would at least work for now.  The point of the checksum is for you to make them available (possibly with signature but we currently don't check that ) so that the script can get your ISO and you checksum and compare (because some users ask for this).  There is no  point whatever in me (or the script) downloading a file, computing a checksum then passing both to a function to see if they check out - I really hope they always will :slightly_smiling_face: ) .   So if you add a file (e.g. SHA256Sums )  with a list of checksums and filenames (or one for each file like ${file}.sha512sum) then the script can be made to work.  For some supported distros we even have to use a checksum file to find the sum and the filename (as it changes mostly daily at some unknown time...), but that works well and is there in the quickget source to re-use.

THope this helps.  Thanks

Reply:
s/THope/Hope/